std_msgs/Header header   
std_msgs/String command
geometry_msgs/Pose pose


## HEADER ------------------------------------------------------------------------
## Standard metadata for higher-level stamped data types.
## This is generally used to communicate timestamped data
## in a particular coordinate frame.

## Two-integer timestamp that is expressed as seconds and nanoseconds.

# builtin_interfaces/Time stamp

## Transform frame with which this data is associated.

# string frame_id

## STRING ------------------------------------------------------------------------

# string data

## POSE --------------------------------------------------------------------------
# # A representation of pose in free space, composed of position and orientation. 
# Point position
# Quaternion orientation

